src/processor/ProfileProcessor.ts

```diff
@@ -1,14 +1,19 @@
 import { fhirdefs } from 'fsh-sushi';
 import { AbstractSDProcessor } from './StructureDefinitionProcessor';
-import { ExportableProfile } from '../exportable';
+import { ExportableProfile, ExportableInvariant } from '../exportable';
 
 export class ProfileProcessor extends AbstractSDProcessor {
-  static process(input: any, fhir: fhirdefs.FHIRDefinitions): ExportableProfile {
+  static process(
+    input: any,
+    fhir: fhirdefs.FHIRDefinitions
+  ): [ExportableProfile, ...ExportableInvariant[]] | [] {
     if (ProfileProcessor.isProcessableStructureDefinition(input)) {
       const profile = new ExportableProfile(input.name);
       ProfileProcessor.extractKeywords(input, profile);
       ProfileProcessor.extractRules(input, profile, fhir);
-      return profile;
+      const invariants = ProfileProcessor.extractInvariants(input);
```

Comment:
As noted in the ExtensionProcessor, if we bump this up a few lines it will allow `InvariantExtractor` to declare its own processed paths.